core/system_menu_method.js

```diff
@@ -241,6 +242,23 @@ function nextArea(callingMenu, formData, extraArgs, cb) {
     );
 }
 
+function setClientEncoding(callingMenu, formData, extraArgs, cb) {
+    // TODO: Also add other encoding types
+    const client = callingMenu.client;
+    let encoding = formData.value.encoding;
+
+    client.log.info(
```

Comment:
This log could probably be moved to `client_term.js` as well since that would capture all paths.

Reply:
Fixed, thanks.